applications/portal/frontend/src/components/Submit/DuplicatedMsg.tsx

```diff
@@ -73,6 +73,7 @@ const DuplicatedMsg = (props) => {
         </Divider>
         <Typography variant="subtitle1" component="p" align="center" sx={styles.message} className="chinese-message">
           此次申请将会被拒绝，因为我们发现您所提交的抗体已经存在于我们的系统中，它的ID是
+          此次申请将会合并，因为我们发现您所提交的抗体已经存在于我们的系统中，它的ID是
```

Comment:
@D-GopalKrishna looks like the new text has been added without removing the old one

Reply:
aha!! thanks @filippomc !!